packages/package-json-lint/src/rules/require-ie10-compatible-dependency-versions.js

```diff
@@ -0,0 +1,41 @@
+const semver = require('semver');
+const maxSatisfying = require('semver/ranges/max-satisfying');
+const { execSync } = require('child_process');
+
+const versionSet = [
+  { name: 'redux', versionRange: '<4.0.0' },
+  { name: 'axios', versionRange: '<=0.18.1' },
```

Comment:
Can this be `<0.19.0` just to stay consistent in using `<` instead of `<=`?

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/713/commits/005b436eb118022969f61a38e5d8a7643357aa77